src/metabase/api/card.clj

```diff
@@ -103,13 +104,15 @@
                                                   [:or
                                                    [:like :c.dataset_query (format "%%card__%s%%" model-id)]
                                                    [:like :c.dataset_query (format "%%#%s%%" model-id)]]]]
-                        :where [:and [:= :m.id model-id] [:not :c.archived]]})
+                        :where [:and [:= :m.id model-id] [:not :c.archived]]
+                        :order-by [[[:lower :c.name] :asc]]})
```

Comment:
merge this with `order-by-name` ?

Reply:
Doesn't quite work since this uses `c.` as the table alias and `order-by-name` doesn't have that